### PR TITLE
Adding attest.md

### PR DESCRIPTION
**Pull request by jvega10**

### CODE REVIEW

File: tdx-attestation-app/ATTEST.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
+# Important Notice as of 10/17
+Support for in-guest attestation is only available using the Azure in-guest app. You can leverage this to attest with Intel Trust Authority. Intel in-guest application is under development and will be released in the coming weeks.
```
Comment (simranparkhe) [inline tdx-attestation-app/ATTEST.md:2]: Mind just removing these top two lines?

Comment (jvega10) [reply tdx-attestation-app/ATTEST.md]: done!

File: tdx-attestation-app/ATTEST.md [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+6. Configure and Build
+```sh
+cmake .
+make
+```
+7. To attest with MAA run the application with the following command:
+```sh
+sudo ./TdxAttest -c maa_config.json
+```
+9. To attest with Intel Trust Authority sign up to retrieve an API Key for [Intel Trust Authority](https://github.com/Azure/azure-compute-tdx-preview/blob/main/ITA.md)
```
Comment (simranparkhe) [inline tdx-attestation-app/ATTEST.md:49]: To attest with Intel Trust Authority, sign up to retrieve an API Key for [Intel Trust Authority] (https://www.intel.com/content/www/us/en/security/trust-authority.html). For more information on Intel Trust Authority see their [documentation](https://docs.trustauthority.intel.com/main/articles/introduction.html).

File: tdx-attestation-app/ATTEST.md [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+``` bash
+cd trustauthority-client/tdx-cli
+```
+
+3. Build the Intel Trust Authority-cli
+``` bash
+make cli
+```
+### Run the Intel Trust Authority-cli
+1. Sign up to retrieve an API Key for [Intel Trust Authority](https://github.com/Azure/azure-compute-tdx-preview/blob/main/ITA.md)
```
Comment (simranparkhe) [inline tdx-attestation-app/ATTEST.md:128]: Sign up to retrieve an API Key for [Intel Trust Authority] (https://www.intel.com/content/www/us/en/security/trust-authority.html). For more information on Intel Trust Authority see their [documentation](https://docs.trustauthority.intel.com/main/articles/introduction.html).

**Comment by michamcr** [comment]
Approved, merging for TDX customers to perform attestation.

**Pull request merged**